### PR TITLE
[docs-only] feat: change the docs to reflect the new deployments

### PR DESCRIPTION
**Pull request by micbar**
## Description

Reflect the recent changes in the docs.

## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: docs/ocis/deployment/continuous_deployment.md [hunk 1]
```diff
@@ -11,67 +11,50 @@ geekdocFilePath: continuous_deployment.md
 
 We are continuously deploying the following deployment examples. Every example is deployed in two flavors:
 
-- Latest: reflects the current master branch state of oCIS and will be updated with every commit to master
-- Daily: reflects the master branch state of oCIS as of the time of deployment
-- Released: reflects the newest release state (currently latest release of version 1) and will be updated with every release
+- Master: reflects the current master branch state of oCIS and will be updated with every commit to master
+- Rolling: reflects the latest rolling release of oCISt and will be updated with every release
```
Comment (ScharfViktor) [inline docs/ocis/deployment/continuous_deployment.md:15]: of oCIS

File: docs/ocis/deployment/continuous_deployment.md [hunk 2]
```diff
@@ -11,67 +11,50 @@ geekdocFilePath: continuous_deployment.md
...
-- oCIS: [ocis.ocis-wopi.daily.owncloud.works](https://ocis.ocis-wopi.daily.owncloud.works)
-- Mail: [mail.ocis-wopi.daily.owncloud.works](https://mail.ocis-wopi.daily.owncloud.works)
+## Rolling Release with OnlyOffice
 
-## Released
+This example is based on the [ocis_full Example](ocis_full.md) and uses the default configuration with OnlyOffice as the office suite.
 
-- oCIS: [ocis.ocis-wopi.released.owncloud.works](https://ocis.ocis-wopi.released.owncloud.works)
-- Mail: [mail.ocis-wopi.released.owncloud.works](https://mail.ocis-wopi.released.owncloud.works)
+- oCIS: [ocis.ocis-onlyoffice.rolling.owncloud.works](https://ocis.ocis-wopi.onlyoffice.owncloud.works)
```
Comment (ScharfViktor) [inline docs/ocis/deployment/continuous_deployment.md:47]: wrong url: 
`[ocis.ocis-onlyoffice.rolling.owncloud.works](ocis.ocis-onlyoffice.rolling.owncloud.works)`

File: docs/ocis/deployment/continuous_deployment.md [hunk 3]
```diff
@@ -11,67 +11,50 @@ geekdocFilePath: continuous_deployment.md
...
 
-# oCIS with latest ownCloud Web
+# oCIS and ownCloud Web with both most recent development versions
 
 Credentials:
 
 - oCIS: see [default demo users]({{< ref "../getting-started#login-to-owncloud-web" >}})
 
 ## Latest
```
Comment (ScharfViktor) [inline docs/ocis/deployment/continuous_deployment.md:55]: master?

File: docs/ocis/deployment/continuous_deployment.md [hunk 4]
```diff
@@ -11,67 +11,50 @@ geekdocFilePath: continuous_deployment.md
...
-# oCIS with WOPI server
-
-Credentials:
-
-- oCIS: see [default demo users]({{< ref "../getting-started#login-to-owncloud-web" >}})
+## Production Release
 
-## Latest
+- oCIS: [ocis.ocis.production.owncloud.works](https://ocis.ocis.production.owncloud.works)
+- Mail: [mail.ocis.production.owncloud.works](https://mail.ocis.production.owncloud.works)
```
Comment (ScharfViktor) [inline docs/ocis/deployment/continuous_deployment.md:36]: could we move it after ## Rolling Release to keep structure: 
Master
Rolling 
Production

File: docs/ocis/deployment/continuous_deployment.md (hunk 1 above)
Comment (micbar) [reply docs/ocis/deployment/continuous_deployment.md:15]: fixed.

File: docs/ocis/deployment/continuous_deployment.md (hunk 3 above)
Comment (micbar) [reply docs/ocis/deployment/continuous_deployment.md:55]: I would not make it explicit.

File: docs/ocis/deployment/continuous_deployment.md [hunk 5]
```diff
@@ -11,67 +11,50 @@ geekdocFilePath: continuous_deployment.md
...
+- Master: reflects the current master branch state of oCIS and will be updated with every commit to master
+- Rolling: reflects the latest rolling release of oCIS and will be updated with every release
+- Production: reflects the latest production release of oCIS and will be updated with every release
 
 The configuration for the continuous deployment can be found in the [oCIS repository](https://github.com/owncloud/ocis/tree/master/deployments/continuous-deployment-config).
 
-# oCIS with Traefik
+# oCIS with Web Office
+
+This deployment is based on our modular [ocis_full Example](ocis_full.md) and uses the default configuration with Collabora Online as the office suite, traefik reverse proxy, cloudimporter and the inbucket mail catching server to showcase the full feature set of oCIS.
```
Comment (mmattel) [inline docs/ocis/deployment/continuous_deployment.md:22]:
```suggestion
This deployment is based on our modular [ocis_full Example](ocis_full.md) and uses the default configuration with traefik reverse proxy, Collabora Online as the office suite, Tika for search and Cloudimporter to showcase the full feature set of oCIS. The Inbucket mail catching server or other services can be easily configured additionally.
```
I have checked the `ocis_full/.env` file to see what is default enabled.

File: docs/ocis/deployment/continuous_deployment.md [hunk 6]
```diff
@@ -82,31 +65,31 @@ Credentials:
...
+- Keycloak admin access: [keycloak.ocis-keycloak.rolling.owncloud.works](https://keycloak.ocis-keycloak.rolling.owncloud.works)
+- Keycloak account management: [keycloak.ocis-keycloak.rolling.owncloud.works/realms/oCIS/account/#/](https://keycloak.ocis-keycloak.rolling.owncloud.works/realms/oCIS/account/#/)
 
-- oCIS: [ocis.ocis-keycloak.released.owncloud.works](https://ocis.ocis-keycloak.released.owncloud.works)
-- Keycloak: [keycloak.ocis-keycloak.released.owncloud.works](https://keycloak.ocis-keycloak.released.owncloud.works)
-- Keycloak account management: [keycloak.ocis-keycloak.released.owncloud.works/realms/oCIS/account/#/](https://keycloak.ocis-keycloak.released.owncloud.works/realms/oCIS/account/#/)
 
 # oCIS with S3 storage backend (MinIO)
 
+This deployment is based on our modular [ocis_full Example](ocis_full.md) and uses a configuration with Collabora Online as the office suite, traefik reverse proxy, cloudimporter and the inbucket mail catching server to showcase the full feature set of oCIS. In addition to that, we deployed a MinIO S3 storage backend. oCIS stores the data in the S3 server and the metadata on the local disk by using the `s3ng` storage driver.
```
Comment (mmattel) [inline docs/ocis/deployment/continuous_deployment.md:77]:
```suggestion
This deployment is based on our modular [ocis_full Example](ocis_full.md), see the [oCIS with Web Office](#ocis-with-web-office) description for the feature set. In addition to that, we deployed a MinIO S3 storage backend. oCIS stores the data in the S3 server and the metadata on the local disk by using the `s3ng` storage driver.
```

File: docs/ocis/deployment/continuous_deployment.md (hunk 5 above)
Comment (micbar) [reply docs/ocis/deployment/continuous_deployment.md:22]: Our config is different

Comment (micbar) [reply docs/ocis/deployment/continuous_deployment.md:22]: I would keep it as it is.

**Review by mmattel — APPROVED** [review]
LGTM 👍

**Pull request merged**